src/components/dropdown/dropdown-item.ts

```diff
@@ -8,7 +10,31 @@ const { prefix } = settings;
  * Dropdown menu item.
  */
 @customElement(`${prefix}-dropdown-item` as any)
-class BXDropdownItem extends LitElement {
+class BXDropdownItem extends FocusMixin(LitElement) {
+  /**
+   * The handle for the `keydown` event on this element.
+   */
+  private _hKeydown: Handle | null = null;
+
+  /**
+   * Handles `keydown` event on this element.
+   */
+  private _handleKeydown = event => {
+    if (event.key === 'ArrowDown' || event.key === 'Down') {
```

Comment:
Utilization of a table used in structured list may make sense here: https://github.com/carbon-design-system/carbon-custom-elements/blob/c69001c/src/components/structured-list/structured-list-row.ts#L13-L21

Reply:
Ah yeah, something like that would work 🤔